ui-integration-test-lib/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+plugins {
```

Comment:
Is your IDE configured differently?  It seems like it is using 2 spaces for indents in kts files.

It makes me wonder if we have some additional config that should be added in our checked in .idea folder

Reply:
Hmm strange. Thank you for pointing it out. I just checked my IDE settings and compared it with the .idea parameters and everything seems alright. I updated the indentation in the `ui-integration-test-lib/build.gradle.kts` and I will keep eye on it for a further work.